sshuttle/client.py

```diff
@@ -159,6 +162,57 @@ def daemon_cleanup():
             raise
 
 
+class ClientMetrics(Metrics):
+
+    def __init__(self):
+        self.dns_request_count = 0
+        self.udp_request_count = 0
+        self.client = StatsClient('localhost', 8125, SSHUTTLE_METRIC_PREFIX + str(os.getpid()))
```

Comment:
the metric endpoint host/port could be passed in through the env instead of hard coded